### PR TITLE
Configure open source feedback

### PR DESCRIPTION
**Pull request by sdwheeler**
Configure open source feedback

### CODE REVIEW

**Comment by sdwheeler** [comment]
Preview URLs

- https://review.learn.microsoft.com/en-us/powershell/azure/what-is-azure-powershell?view=azps-10.3.0&branch=pr-en-us-2724&preview=open-source-feedback
- https://review.learn.microsoft.com/en-us/powershell/azure/faq?view=azps-10.3.0&branch=pr-en-us-2724&preview=open-source-feedback
- https://review.learn.microsoft.com/en-us/powershell/module/az.accounts/clear-azcontext?view=azps-10.3.0&branch=pr-en-us-2724&preview=open-source-feedback

File: .github/ISSUE_TEMPLATE/03-customer-feedback.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+name: "📄 Feedback control template | Do not use"
+title: "[Documentation feedback] - "
```
Comment (sdwheeler) [inline .github/ISSUE_TEMPLATE/03-customer-feedback.yml:2]:
```suggestion
title: " "
```

File: docfx.json [hunk 2]
```diff
@@ -583,10 +585,33 @@
...
+                "azps-10.4.1/**/*": "https://learn.microsoft.com/powershell/azure/azps-docs-style-guide",
+                "docs-conceptual/**/*": "https://learn.microsoft.com/powershell/azure/azps-docs-style-guide"
+            },
+            "open_source_feedback_issueUrl": {
+                "azps-10.4.1/**/*": "https://github.com/Azure/azure-powershell/blob/main/.github/ISSUE_TEMPLATE/4-AZ-DOC-BUG.yml",
+                "docs-conceptual/**/*": "https://github.com/microsoftdocs/azure-docs-powershell/issues/new?template=03-customer-feedback.yml"
+            },
+            "open_source_feedback_issueTitle": {
+                "azps-10.4.1/**/*": "Customer docs feedback - ",
+                "docs-conceptual/**/*": "Customer docs feedback - "
```
Comment (sdwheeler) [inline docfx.json:598]:
```suggestion
                "azps-10.4.1/**/*": "",
                "docs-conceptual/**/*": ""
```

**Review by sdwheeler — COMMENTED** [review]
Need to remove the title contents to avoid spam issues.

File: .github/ISSUE_TEMPLATE/03-customer-feedback.yml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+description: >-
+  This template is intended for use by the feedback control on the bottom of every page on the live
+  site. If you aren't using the feedback control, choose one of the other templates.
```
Comment (sdwheeler) [inline .github/ISSUE_TEMPLATE/03-customer-feedback.yml:5]:
```suggestion
name: "📄 Feedback control template"
title: "Feedback"
description: >-
  ⛔ This template is intended for use by the feedback control on the bottom of every page on the
  live site. If you aren't using the feedback control, choose one of the other templates.⛔
```

**Comment by mikefrobbins** [comment]
Issue control template for conceptual docs has been merged via PR #2753

**Comment by mikefrobbins** [comment]
PR Azure/azure-powershell#22899 containing the Issue control template for reference docs must be merged before this PR goes live with the new docs feedback system in this repo.

**Review by mikefrobbins — DISMISSED** [review]
LGTM

**Pull request merged**